project/android/build.gradle

```diff
@@ -1,21 +1,23 @@
 buildscript {
     ext.kotlin_version = '1.5.0'
     repositories {
-        google()
         mavenCentral()
+        google()
+       
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.2.1'
+        classpath 'com.android.tools.build:gradle:7.0.3'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
         classpath 'com.google.gms:google-services:4.3.8'
     }
 }
 
 allprojects {
     repositories {
-        google()
```

Comment:
Please remove extra spaces and it is better to keep the gradle version but 7.0.3 works so it is fine.